Demos/Makefile

```diff
@@ -6,7 +6,15 @@ test:	all
 	python run_numeric_demo.py
 	python run_spam.py
 	cd callback; $(MAKE) test
-
+	cd embed; $(MAKE) test
+	#
+	# longer running
+	#python integrate_timing.py
+	# also  run
+	# python setup.py build_ext --inplace in
+	#  benchmarks to build benchmarks
+	# python setup.py build_ext --inplace in libraries then
```

Comment:
The line breaks look misplaced here. Commands to copy should always be in a separate line to avoid confusion.